common/static/common/js/discussion/main.js

```diff
@@ -2,10 +2,9 @@
           DiscussionUser, DiscussionUserProfileView, DiscussionUtil */
 (function() {
     'use strict';
-    var DiscussionApp, DiscussionProfileApp;
 
     if (typeof Backbone !== "undefined" && Backbone !== null) {
-        DiscussionApp = {
+        window.DiscussionApp = {
```

Comment:
Why is this needed? Would prefer to keep things off of `window` unless totally unavoidable.